templates/implementation_meta_issue.txt

```diff
@@ -28,3 +28,4 @@ The implementation plan for this issue is <link to implementation plan>.
 - [ ] Issue for documentation:
 - [ ] Issue for specification:
 - [ ] Issue for code cleanup:
+- [ ] Issue for validating behavior of codegen:
```

Comment:
This sounds pretty nebulous. Can we word it more concretely?

How about:

```
- [ ] Issue to validate that build package handles feature:
- [ ] Issue to validate that AngularDart template compiler handles feature:
```

?

Reply:
We have an existing entry for Angular above.  Should we just add package build as an additional "Implementation issue" to be filed above?

Reply:
They don't need to be separate. Updated pr to add "build system" wording next to Angular.